core/src/epicli/data/common/ansible/playbooks/roles/postgres_exporter/meta/main.yml

```diff
@@ -0,0 +1,3 @@
+---
+dependencies:
+  - role: preflight_facts
```

Comment:
Are you sure that preflight_facts will be used in your role?

Reply:
preflight facts is started at the beginning inside epiphany code and not related to this role. So 100% it is used.